src/app.js

```diff
@@ -71,7 +71,6 @@ app.use('/user', usersRouter)
 
 // Error handlers
 // Use HB before all other error handlers.
```

Comment:
@peetucket this comment should be removed,no?

Reply:
yes, indeed, good catch  ... Justin got it in a different PR: https://github.com/LD4P/sinopia_api/pull/159/files#diff-c72a907ac323cd2f334ed0e2bd07d15ab62581c4753660c8a0d1c681b30be4b6L73